Sources/BraveWallet/Crypto/AssetIconView.swift

```diff
@@ -68,8 +65,24 @@ struct AssetIconView: View {
       }
     }
     .frame(width: length, height: length)
+    .overlay(tokenLogo, alignment: .bottomTrailing)
     .accessibilityHidden(true)
   }
+  
+  private var networkNativeTokenLogo: UIImage? {
+    if let logo = network.nativeTokenLogo {
+      return UIImage(named: logo, in: .module, with: nil)
+    }
+    return nil
+  }
+  
+  @ViewBuilder private var tokenLogo: some View {
+    if !(token.isNft || token.isErc721), !network.isNativeAsset(token), let image = networkNativeTokenLogo {
+      Image(uiImage: image)
+        .resizable()
+        .frame(width: 15, height: 15)
+    }
+  }
```

Comment:
can we also add tokenLogo for NFTs?

Reply:
Yeah nice catch, thought they weren't shown on NFTs for some reason 🤔. I'll update this to add to NFTs.

Reply:
Fixed in [67e9c3b](https://github.com/brave/brave-ios/pull/6451/commits/67e9c3b377c5319a90b9547a0299f910e413fe69)